demisto_sdk/__main__.py

```diff
@@ -208,22 +208,31 @@ def unify(**kwargs):
     '-a', '--validate-all', is_flag=True, show_default=True, default=False,
     help='Whether to run all validation on all files or not'
 )
+@click.option(
+    '-i', '--input', help='The path of a pack to validate specifically.'
```

Comment:
i dont think we need another option for that, lets use `-p`, otherwise it will just be confusing

Reply:
A lot of our commands, using -i as input. It just to keep it consistent. 
Change anyway? 
Also the -p exists for specific file path

Reply:
i just dont want to have both
i agree we should stick to `-i`, so lets set `-p` to `hidden=True` and `-i` for all kind of inputs: files, packs, etc